bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tippy from 'tippy.js';
```

Comment:
remove extension on imports

Reply:
tippy.js is the package name

Reply:
> `import Tippy from 'tippy.js';`
`import Tippy from 'tippy';` - we can omit .js / .scss / etc when importing

Reply:
In this case, i can't because tippy.js is the package name.